compiler/resolution/callDestructors.cpp

```diff
@@ -1065,22 +1065,57 @@ void insertReferenceTemps(CallExpr* call) {
 // flag is ever called and raises an error if so.
 static void checkForErroneousInitCopies() {
 
+  // Propagate uses into autoCopy/initCopy fns
```

Comment:
This should clarify what 'uses' are being propagated.  e.g. "Propagate uses of FLAG_ERRONEOUS_INITCOPY/FLAG_ERRONEUS_AUTOCOPY into init/autocopy functions that call erroneous init/autocopy functions."